mreg_cli/main.py

```diff
@@ -155,10 +155,13 @@ def main():
             str(config.get("url")),
             fail_without_token=args.token_only,
         )
-
     except (EOFError, KeyboardInterrupt, LoginFailedError) as e:
-        print(e)
+        if isinstance(e, LoginFailedError):
```

Comment:
Is there a reason not to override `__str__` specifically for LoginFailedError to do `print_self()`?

Reply:
Monday and too much Rust. Ignore me. :)